Resources/doc/events.rst

```diff
@@ -11,155 +11,71 @@ see Doctrine's `Event Documentation`_.
     document managers tied to that connection. Listeners and subscribers may be
     registered with all event managers or just one (using the connection name).
 
-In Symfony, you can register a listener or subscriber by creating a service
-and then `tagging`_ it with a specific tag.
+To register a service to act as an event listener you have to tag it with the
+``doctrine_mongodb.odm.event_listener`` tag.
 
-Event Listeners
----------------
-
-Use the ``doctrine_mongodb.odm.event_listener`` tag to
-register a listener. The ``event`` attribute is required and should denote
-the event on which to listen. By default, listeners will be registered with
-event managers for all connections. To restrict a listener to a single
-connection, specify its name in the tag's ``connection`` attribute.
-
-The ``priority`` attribute, which defaults to ``0`` if omitted, may be used
-to control the order in which listeners are registered. Much like Symfony's
-`event dispatcher`_, greater number will result in the listener executing
-first and listeners with the same priority will be executed in the order that
-they were registered with the event manager.
-
-Lastly, the ``lazy`` attribute, which defaults to ``false`` if omitted, may
```

Comment:
The `lazy` option is now undocumented. I'd deem the `priority` part valuable as well, I for one never remember which comes first 10 or -10 :)